BraveShared/Analytics/UserReferralProgram.swift

```diff
@@ -78,6 +79,12 @@ public class UserReferralProgram {
             
             completion(nil)
         }
+        
+        if let refCode = refCode {
+            Preferences.URP.referralCode.value = refCode
```

Comment:
This pref is not new right? Is this the only place we can set this?

Reply:
Not new, and no. This only gets set for ref codes passed in via clipboard. 
I'll add comment here too.

It also gets set if the server passes down a ref code.